src/dbnode/storage/block/block.go

```diff
@@ -55,7 +55,6 @@ type dbBlock struct {
 
 	mergeTarget DatabaseBlock
 
-	retriever  DatabaseShardBlockRetriever
 	retrieveID ident.ID
```

Comment:
Can you rename `retrieveID` since it's not used for retrieving any more, it's just used for accounting purposes, mainly in the wired list.

So just rename it to `seriesID` here and in the `wiredListEntry` it should also be renamed to `seriesID`.  

That sounds good?